encode-go_1_8.go

```diff
@@ -17,23 +17,33 @@
 //
 // Copyright holder is ArangoDB GmbH, Cologne, Germany
 //
-// Author Ewout Prangsma
-//
 
 //go:build go1.8
```

Comment:
We probably can drop the support for go1.8 already - I think current go-driver will not compile with this version anyway.